vendor.conf

```diff
@@ -61,25 +61,25 @@ github.com/docker/docker 86f080cff0914e9694068ed78d503701667c4c00
 github.com/docker/spdystream 449fdfce4d962303d702fec724ef0ad181c92528
 github.com/emicklei/go-restful v2.9.5
 github.com/google/gofuzz v1.0.0
-github.com/json-iterator/go v1.1.7
+github.com/json-iterator/go v1.1.8
 github.com/modern-go/reflect2 1.0.1
 github.com/modern-go/concurrent 1.0.3
 github.com/opencontainers/selinux 5215b1806f52b1fcc2070a8826c542c9d33cd3cf
 github.com/seccomp/libseccomp-golang v0.9.1
 github.com/tchap/go-patricia v2.2.6
 golang.org/x/crypto 69ecbb4d6d5dab05e49161c6e77ea40a030884e1
 golang.org/x/oauth2 0f29369cfe4552d0e4bcddc57cc75f4d7e672a33
-golang.org/x/time 85acf8d2951cb2a3bde7632f9ff273ef0379bcbd
-gopkg.in/inf.v0 v0.9.0
-gopkg.in/yaml.v2 v2.2.2
-k8s.io/api kubernetes-1.16.0-rc.2
-k8s.io/apimachinery kubernetes-1.16.0-rc.2
-k8s.io/apiserver kubernetes-1.16.0-rc.2
-k8s.io/cri-api kubernetes-1.16.0-rc.2
-k8s.io/client-go kubernetes-1.16.0-rc.2
-k8s.io/klog v0.4.0
-k8s.io/kubernetes v1.16.0-rc.2
-k8s.io/utils c2654d5206da6b7b6ace12841e8f359bb89b443c
+golang.org/x/time 9d24e82272b4f38b78bc8cff74fa936d31ccd8ef
+gopkg.in/inf.v0 v0.9.1
+gopkg.in/yaml.v2 v2.2.3
```

Comment:
from https://github.com/containerd/cri/blob/release/1.3/vendor.conf

using gopkg.in/yaml.v2 v2.2.4

Reply:
yes, would be good to update it here as well

Reply:
👍